components/Datepicker/src/index.tsx

```diff
@@ -0,0 +1,366 @@
+        >
+          {date.getDate()}
+        </button>
+      );
+    }
+    return '';
+  };
+
+  return (
+    <div className={rootStyles} ref={ref}>
+      <input
+        disabled={props.disabled}
+        type="date"
+        className="denhaag-datepicker__input"
+        ref={inputRef}
+        value={state.selected ? formatISO(state.selected, { representation: 'date' }) : ''}
+        onChange={(event) => {
+          setState({
+            ...state,
+            current:
+              event.target.value && new Date(event.target.value).getDate()
+                ? new Date(event.target.value)
+                : state.current,
+            selected:
+              event.target.value !== undefined
+                ? new Date(event.target.value).getDate()
+                  ? new Date(event.target.value)
+                  : state.selected
+                : undefined,
+          });
+        }}
+        onClick={(event) => {
+          event.preventDefault();
+          setState({ ...state, opened: true });
+        }}
+        onKeyDown={onKeyDownInput}
+      />
+      <button
+        className="denhaag-datepicker__button"
+        onClick={() => {
+          setState({ ...state, opened: !state.opened });
+        }}
+        aria-label={openCalendarLabel}
+      ></button>
+      <div
+        className={clsx('denhaag-datepicker__calendar', { 'denhaag-datepicker__calendar--hidden': !state.opened })}
+        role="dialog"
+        aria-modal="true"
+        aria-labelledby="denhaag-datepicker-calendar-month"
+        aria-live="polite"
+      >
+        <div className="denhaag-datepicker__calendar-header">
+          <button
+            type="button"
+            className="denhaag-datepicker__calendar-previous"
+            onClick={() => {
+              setState({ ...state, current: subMonths(state.current, 1) });
+            }}
+            ref={backButtonRef}
+            onKeyDown={onKeyDownBack}
+            aria-label={previousMonthLabel}
+          ></button>
+          <span
+            id="denhaag-datepicker-calendar-month"
+            className="denhaag-datepicker__calendar-month"
+            aria-live="polite"
+          >
+            <time dateTime={format(state.current, 'yyyy-MM')}>
+              {format(state.current, 'MMMM yyyy', { locale: locale })}
+            </time>
+          </span>
+          <button
+            type="button"
+            className="denhaag-datepicker__calendar-next"
+            onClick={() => {
+              setState({ ...state, current: addMonths(state.current, 1) });
+            }}
+            onKeyDown={onKeyDownNext}
+            aria-label={nextMonthLabel}
+          ></button>
+        </div>
+        <table
+          className="denhaag-datepicker__calendar-table"
+          role="grid"
+          aria-labelledby="denhaag-datepicker-calendar-month"
+        >
+          <tbody>
+            <tr>
+              {Array.from(Array(7)).map((_, i) => (
+                <th
```

Comment:
I think we should give the `<th>` a `scope=col` attribute here to indicate its a column header

Reply:
Fixed!